errands-server.go

```diff
@@ -215,7 +216,7 @@ func (s *ErrandsServer) createAPI() {
 	// Singular errand Routes:
 	s.ErrandRoutes = s.API.Group("/v1/errand")
 	// Get an errand by id:
-	s.ErrandRoutes.GET("/:id", s.createErrand)
+	s.ErrandRoutes.GET("/:id", s.getErrand)
```

Comment:
oof...I'm a little worried there's something out there actually using this path to `create` an errand....but we'll just have to wait and see. Be on the lookout for failing crons